samples/nextjs/package.json

```diff
@@ -52,7 +52,7 @@
   "devDependencies": {
     "@graphql-codegen/cli": "^1.19.1",
     "@graphql-codegen/plugin-helpers": "^1.18.1",
-    "@graphql-codegen/typescript": "^1.17.11",
+    "@graphql-codegen/typescript": "^1.21.1",
```

Comment:
@ambrauer Is graphql-let working fine if you update only this package?

Reply:
@illiakovalenko I'm not sure what you mean by "only"? I will double check, but graphql-let should be working fine.

Reply:
Cool, I mean that we have another `@graphql-codegen` dependencies. If it's working - it's good.